gravatar-ui/build.gradle.kts

```diff
@@ -50,8 +51,13 @@ android {
     buildFeatures {
         compose = true
     }
-    composeOptions {
-        kotlinCompilerExtensionVersion = "1.5.8"
+    composeCompiler {
+        // Enable 'strong skipping'
+        // https://medium.com/androiddevelopers/jetpack-compose-strong-skipping-mode-explained-cbdb2aa4b900
+        enableStrongSkippingMode.set(true)
+        // Needed for Layout Inspector to be able to see all of the nodes in the component tree:
+        // https://issuetracker.google.com/issues/338842143
+        includeSourceInformation.set(true)
```

Comment:
❓Do you suffer from the issue mentioned [here](ttps://issuetracker.google.com/issues/338842143)? I've tried removing this for both (library and app), and it works well for me. So I wonder if it is still necessary.

Reply:
> Thanks for the reply! That was indeed the fix. Looks like the new Compose Compiler DSL doesn't have this turned on by default.
>
> credit: https://issuetracker.google.com/issues/338842143#comment3

Actually no I didn't seem to suffer it but I turned it on because of the comment above. It seems it's a flag expected to be enabled [1] but the new compose compiler dsl has it disabled by default.

[1]: https://issuetracker.google.com/issues/338842143#comment2

Reply:
IMO it's a harmless flag when turned on though, so I'd leave it on as it seems to fix a potential issue and there are plans in the future to warn users when it's turned off. Let me know if you want it removed.